DataFormats/Detectors/TRD/src/RawData.cxx

```diff
@@ -9,13 +9,170 @@
 // or submit itself to any jurisdiction.
 
 #include <iostream>
-//#include "DataFormatsTRD/RawData.h"
+#include "DataFormatsTRD/RawData.h"
 
 namespace o2
 {
 
 namespace trd
 {
 
+uint32_t unpacklinkinfo(const HalfCRUHeader& cruhead, const uint32_t link, const bool data = true)
+{
+    //TODO this needs some more ellaborate testing. I got the packing to work so for now,
+    //     its a question of unpacking info in line 88 and 89.
```

Comment:
This should also be documented, i.e. what goes into the uint32_t and at what bit positions. I

Reply:
fixed in 7dac991d840